File: app/src/release/java/com/soumya/wwdablu/zomatobuddy/BuddyReleaseTree.java
package com.soumya.wwdablu.zomatobuddy;

import android.util.Log;

import timber.log.Timber;

public class BuddyReleaseTree extends Timber.Tree {

    @Override
    protected boolean isLoggable(String tag, int priority) {

        switch (priority) {

            case Log.VERBOSE:
            case Log.DEBUG:
            case Log.INFO:
                return false;
        }

        return super.isLoggable(tag, priority);
    }

    @Override
    protected void log(int priority, String tag, String message, Throwable t) {

        super.log(priority, tag + " : " + message, t);
    }
}

File: app/src/release/java/com/soumya/wwdablu/hungry/BuddyReleaseTree.kt
package com.soumya.wwdablu.zomatobuddy

import android.util.Log

class BuddyReleaseTree : Timber.Tree() {
    @Override
    protected fun isLoggable(tag: String?, priority: Int): Boolean {
        when (priority) {
            Log.VERBOSE, Log.DEBUG, Log.INFO -> return false
        }
        return super.isLoggable(tag, priority)
    }

    @Override
    protected fun log(priority: Int, tag: String?, message: String?, t: Throwable?) {
        super.log(priority, tag.toString() + " : " + message, t)
    }
}